dotnetapp/Dockerfile.build

```diff
@@ -1,8 +1,8 @@
-FROM microsoft/dotnet:1.0-sdk-projectjson
+FROM microsoft/dotnet:1.0-sdk-msbuild
```

Comment:
This sample is using LTS.  I am not sure there is a reason to use current for this sample.